runtest

```diff
@@ -11,4 +12,5 @@ then
     echo "You need tcl 8.5 or newer in order to run the Redis test"
     exit 1
 fi
+$MAKE -C tests/modules && \
```

Comment:
for some reason, the test modules are not part of the redis build on purpose.
i think the reason was that Antirez didn't want any production build to fail, or tests to fail due to module tests.
while i don't agree with it, i don't think we wanna change this as part of this PR.